lib/datastore.js

```diff
@@ -84,7 +114,8 @@ class DataStore {
    * See {@link datastore.create} for the details of what `{cfg}`
    * parameters are supported.
    *
-   * @param {Onject} cfg - The configuration parameters
+   * @param {Object} cfg - The configuration parameters.
```

Comment:
Nit: The consensus seems to be no hyphen and no punctuation. I don't have a preference for how we do this, but consistency would be nice.